docs/clients/grpc/observability.md

```diff
@@ -0,0 +1,106 @@
+# Observability
+
+The EventStoreDB gRPC clients are designed with observability in mind, offering support for OpenTelemetry. This integration provides a set of distributed traces, enabling developers to gain deeper insights into their system.
+
+::: warning
+Currently not all clients have support for observability using OpenTelemetry.
+:::
+
+::: note
+Only the append and subscribe (Catchup and Persistent) operations are instrumented.
```

Comment:
I do not understand what this sentence is trying to say. Also, I think it can be combined with the note box above so we're not putting 2 boxes in a row.

Reply:
I've combined it into a single section. Only the append and subscribe methods are instrumented. Other operations, such as reading and projections, do not emit traces atm.